rpi-imager-haos.json

```diff
@@ -16,7 +16,7 @@
       "name": "Home Assistant OS 10.5 (RPi 3)",
       "description": "Open source home automation that puts local control and privacy first.",
       "url": "https://github.com/home-assistant/operating-system/releases/download/10.5/haos_rpi3-64-10.5.img.xz",
-      "icon": "https://version.home-assistant.io/rpi-imager-haos.png",
+      "icon": "https://version.home-assistant.io/brand/logo.png",
```

Comment:
Do we know how often this is synced?
Removing the current image will break it for everyone until they fetch the new data.

Reply:
🤷 Maybe @agners knows?

Reply:
I think this is fetched from clients directly when they browse that menu.